src/dashboard/Data/Browser/DataBrowser.react.js

```diff
@@ -426,26 +505,69 @@ export default class DataBrowser extends React.Component {
     return (
       <div>
-        <BrowserTable
-          appId={applicationId}
-          order={this.state.order}
-          current={this.state.current}
-          editing={this.state.editing}
-          simplifiedSchema={this.state.simplifiedSchema}
-          className={className}
-          editCloneRows={editCloneRows}
-          handleHeaderDragDrop={this.handleHeaderDragDrop}
-          handleResize={this.handleResize}
-          setEditing={this.setEditing}
-          setCurrent={this.setCurrent}
-          setCopyableValue={this.setCopyableValue}
-          setContextMenu={this.setContextMenu}
-          onFilterChange={this.props.onFilterChange}
-          onFilterSave={this.props.onFilterSave}
-          selectedCells={this.state.selectedCells}
-          handleCellClick={this.handleCellClick}
-          {...other}
-        />
+        <div>
+          <BrowserTable
+            appId={applicationId}
+            order={this.state.order}
+            current={this.state.current}
+            editing={this.state.editing}
+            simplifiedSchema={this.state.simplifiedSchema}
+            className={className}
+            editCloneRows={editCloneRows}
+            handleHeaderDragDrop={this.handleHeaderDragDrop}
+            handleResize={this.handleResize}
+            setEditing={this.setEditing}
+            setCurrent={this.setCurrent}
+            setCopyableValue={this.setCopyableValue}
+            selectedObjectId={this.state.selectedObjectId}
+            setSelectedObjectId={this.setSelectedObjectId}
+            callCloudFunction={this.props.callCloudFunction}
+            setContextMenu={this.setContextMenu}
+            onFilterChange={this.props.onFilterChange}
+            onFilterSave={this.props.onFilterSave}
+            selectedCells={this.state.selectedCells}
+            handleCellClick={this.handleCellClick}
+            isPanelVisible={this.state.isPanelVisible}
+            panelWidth={this.state.panelWidth}
+            isResizing={this.state.isResizing}
+            {...other}
+          />
+          {this.state.isPanelVisible && (
+            <ResizableBox
+              width={this.state.panelWidth}
+              height={Infinity}
+              minConstraints={[400, Infinity]}
+              maxConstraints={[this.state.maxWidth, Infinity]}
+              onResizeStart={this.handleResizeStart}  // Handle start of resizing
+              onResizeStop={this.handleResizeStop}    // Handle end of resizing
+              onResize={this.handleResizeDiv}
+              resizeHandles={['w']}
+              className={styles.resizablePanel}
+              style={
+                {
+
+                  position: 'fixed',
+                  top: '96px',
+                  right: '0',
+                  bottom: '0px',
+                }
+              }
+            >
+              <div style={{
+                height: '100%',
+                overflow: 'auto',
+                display: 'flex',
```

Comment:
I am not sure if this was intended, but now, because of that line, each panel segment is rendered as a separated column.

<img width="765" alt="image" src="https://github.com/user-attachments/assets/3333ecea-9a08-4722-a363-6a718b644e6d">

Reply:
sorry my bad , it was a mistake from my side 
I will fix it asap

Reply:
If you need flex, then setting `flex-direction` to `column` might do the trick.